gc/gc-base/src/main/java/org/projectnessie/gc/base/DistributedIdentifyContents.java

```diff
@@ -87,18 +92,32 @@ private static int getPartitionsCount(GCParams gcParams, List<Reference> referen
   }
 
   private static Map<String, ContentBloomFilter> mergeLiveContentResults(
-      List<Map<String, ContentBloomFilter>> bloomFilterMaps) {
+      List<Map<String, ContentBloomFilter>> bloomFilterMaps, double bloomFilterFpp) {
+    Set<String> mergedContentIds = new HashSet<>();
     Map<String, ContentBloomFilter> output = new HashMap<>();
     bloomFilterMaps.forEach(
         map ->
             map.forEach(
                 (k, v) -> {
                   if (output.containsKey(k)) {
                     output.get(k).merge(v);
+                    mergedContentIds.add(k);
                   } else {
                     output.put(k, v);
                   }
                 }));
+    // Since we merged bloom filters log in case their quality deteriorated
+    mergedContentIds.stream()
+        .forEach(
+            contentId -> {
+              double fpp = output.get(contentId).getExpectedFpp();
+              if (fpp > bloomFilterFpp) {
+                LOGGER.info(
```

Comment:
a) good to have log level `WARN` for this.

b) Have we explored a testcase which merges equal size bloom filters and changes its fpp after merge? may be we can add one in `TestContentBloomFilter`.  It should validate whether fpp changed after merge.

Reply:
i went with INFO because being slightly above the target could still be fine.

https://github.com/google/guava/blob/a0e2577de61a0d7e8a3dd075be66a31c93ea0446/guava/src/com/google/common/hash/BloomFilter.java#L177-L188

mentions only a significantly higher value would indicate a problem.
maybe we can put it on WARN or ERROR if it is above certain tresholds.